.github/workflows/codebuild-ci.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v1
         with:
           role-to-assume: ${{ secrets.CI_AWS_ROLE_ARN }}
+          role-duration-seconds: 120
```

Comment:
Shouldn't this value be >= 3600 to map it to an hour?

Reply:
Per the docs the default value is 1 hour - https://github.com/aws-actions/configure-aws-credentials#assuming-a-role

Reply:
Fixed